.gitlab-ci.yml

```diff
@@ -8,14 +8,17 @@ variables:
   PROJECT_ALLOC_NAME: ${CI_PROJECT_NAME}_ci_${CI_PIPELINE_ID}
   BUILD_ROOT: ${CI_PROJECT_DIR}
   ACCOUNT: -p pdebug -A eng
+  L_ACCOUNT: -q pdebug -G eng
```

Comment:
Now that we have two, let’s rename the variables:
I think `SLURM_ACCOUNT` and `LSF_ACCOUNT` would be more appropriate.